src/main/java/org/zalando/planb/revocation/util/UnixTimestamp.java

```diff
@@ -1,14 +1,23 @@
 package org.zalando.planb.revocation.util;
 
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
 import java.util.Date;
 
-public class UnixTimestamp {
+public abstract class UnixTimestamp {
+
+    private UnixTimestamp() {
+    }
 
     public static int now() {
-        return (int) (System.currentTimeMillis() / 1000);
+        return internalToSeconds(System.currentTimeMillis());
     }
 
     public static int of(Date date) {
-        return (int) (date.getTime() / 1000);
+        return internalToSeconds(date.getTime());
+    }
+
+    protected static int internalToSeconds(long millis) {
```

Comment:
Can we call the method "millisToSeconds" instead? I think that would be more appropriate.